runtime/makelib/targets.mk.linux.inc.ftl

```diff
@@ -539,6 +539,10 @@ MHInterpreterFull$(UMA_DOT_O) : MHInterpreterFull.cpp
 MHInterpreterCompressed$(UMA_DOT_O) : MHInterpreterCompressed.cpp
 	$(PPC_GCC_CXX) $(PPC_GCC_CXXFLAGS) -c $<
 
+# special handling bcverify.c for unexpected crash due to over-optimization in compilation
+bcverify$(UMA_DOT_O) : bcverify.c
+	$(CC) $(CFLAGS) -O2 -c -o $@ $<
```

Comment:
I don't think this should be inside the USE_PPC_GCC check, it is unrelated. Also, it should be restricted to plinux LE, unless the problem has been seen on BE (which uses a different compiler).

Reply:
While the problem and fix is related to using gcc, that isn't what the USE_PPC_GCC flag is about. It's about using gcc to compile the above files even when the main compiler is different (xlc).

Reply:
Agreed and updated the setting against the suggestion above.